### ISSUE TITLE
Stuck during init process

### ISSUE THREAD

**Issue by bonndan** [talentsconnect/braumeister#34]
Hi,

I have issues setting up braumeister at a customer. After having entered my username the process is stuck. Not even Ctrl-C helps. 

<img width="494" alt="image" src="https://user-images.githubusercontent.com/143695/211285579-6e637fd5-5b1f-4ad2-a2a0-336a20bb526a.png">

The Jira instance is on-prem and my username is shown as "Pozzi" when being logged in, or "pozzi" in the login form. I'm using Python 3.11.1 from Git Bash on Windows. Any chance to see debug ouput?

**Comment by Rotzlucky** [comment]
There is a verbose option. But it's only in affect after the initialisation.

You could set up the .braumeister config with any editor, if that helps!
```

[general]
verbose = false

[git]
main_branch_name = main

[jira]
url = https://your-domain.atlassian.net
username = my-user
password = my-api-token
destination_transition = Merged
branch_custom_field_id = customfield_5711
```

**Comment by bonndan** [comment]
Yes, that helps. We will perhaps use braumeister with several repos and would have repeat the above procedure. If you're interested in debugging the problem, tell us how and we'll provide feedback.

Thanks!

**Comment by Rotzlucky** [comment]
The code gets stuck in here
https://github.com/talentsconnect/braumeister/blob/master/Braumeister/setup.py

Going from your screenshot you reached line 98 but not 103.

Braumeister is just collecting at that moment! 
Writing to disk will happen later from line 135ff. So it should not be a problem with file permissions.

I would probably just play around with some print statements to try to debug that. Feel free to fork and try it yourself.

**Comment by Rotzlucky** [comment]
Another thought:

The next question that is failing for you is in regards to password.
So it's the only question that calls the `ask_required` method with a value of `True` for the secure parameter in order to obfuscate the entered password.
Maybe the lib getpass we are using there behaves differently on Git Bash

**Comment by bonndan** [comment]
I have tried braumeister in PowerShell and it worked flawlessly. Seems indeed to be a problem with Git Bash. Other than running braumeister as installed with pip exceeds my time and knowledge, so I won't fork it. If there is a way to run a python program with debug output, let me know.

**Comment by Rotzlucky** [comment]
Ok, thanks for confirming! When I have any more input I let you know!
Will leave the issue open. Maybe we will get an opportunity to reproduce it!
